entrypoint.sh

```diff
@@ -41,6 +41,11 @@ add_mask "${USER_ACCESS_TOKEN}"
 {
   mkdir $TEMP_CLONE_WIKI_FOLDER
   cd $TEMP_CLONE_WIKI_FOLDER || exit 1
+
+  # Add Safe Directory to initialize sub repo
+  git config --global --add safe.directory "/github/workspace"
```

Comment:
Just wondering, is this line necessary? 🙏

Reply:
Ah, I'll create a chore to investigate if this is necessary 👍